server/views/pages/bookedtoday/summaryWithRecord.njk

```diff
@@ -80,7 +78,7 @@
                 <div class='govuk-grid-column-one-third'>
                     <div class="profile-align-image">
                         <img
-                            src="{{ '/prisoners/' + arrival.prisonNumber + '/image' if arrival.prisonNumber else '/assets/images/placeholder-image.png' }}"
+                            src="{{ '/prisoners/' + singleMatch.prisonNumber + '/image' if singleMatch.prisonNumber else '/assets/images/placeholder-image.png' }}"
```

Comment:
no need for conditional?

Reply:
true

Reply:
will merge this and fix on the next pr - hope that's ok even though it's for the next ticket

Reply:
removed in last commit